content/gui/xml/mainmenu/settings.xml

```diff
@@ -101,7 +101,7 @@
 			</HBox>
 
 			<Slider size="300,20" name="mousesensitivity" is_focusable="0"
-				orientation="0" scale_start="-0.1" scale_end="0.5" step_length="1.0" />
+				  orientation="0" scale_start="0.1" scale_end="5.0" step_length="0.25" />
```

Comment:
So this changes the maximum value of the setting internally from 0.5 to 5.0. 

The FIFE code mentions a allowed range: 

https://github.com/fifengine/fifengine/blob/e875cedd88d5860e3f9d03af2b8bf86771e7c07f/engine/core/eventchannel/eventmanager.h#L133

```cpp
/** Sets mouse sensitivity
* The sensitivity is limited to the range -0.99 - 10.0.
*/
void setMouseSensitivity(float sensitivity);
```

Would it make sense to allow the whole range?